### ISSUE TITLE
Umbraco Forms 8.11.0 Can´t access entries from the Forms dashboard 

### ISSUE THREAD

**Issue by Aaen** [umbraco/Umbraco.Forms.Issues#753]
When you have submitted a form and are trying to see the entry then you just get redirected to the form itself 

### Bug summary

When you are trying to view your entires on the form in the backoffice then it just goes to the form itself. Also the entires folder in the forms tree is missing 

### Specifics
A fresh Umbraco install with

Umbraco 8.18.0
Forms 8.11.0
Deploy 4.5.0.0

### Steps to reproduce
1. Set up and Umbraco installaiton
2. Install Umbraco Forms 8.11.0
3. Set up a form 
4. Submit the form 
5. Visit the entry from the Umbraco Forms dashboard

### Expected result
I would expect to get to the overview of the entries that are submitted in the form 

### Actual result
It just go to the form itself

**Comment by Aaen** [comment]
As a workaround for now 

Check if the user account you are accessing the CMS with has access to "View entries" 

Under Users > Form Security.

**Comment by AndyButland** [comment]
We introduced this split of permission in 8.11 - so a user can have access to form design, or viewing entries, or both.  Previously if you had access to both or neither.

For _upgrades_ we made sure that if you had access to forms, you would get both permissions.

But for _new installs_ I guess we didn't do that.  So you only get from design by default.  I think we should have the "view entries" permission by default too - so we have the same behaviour as before - so I'll look to fix that for the next release.

Workaround for now is just to set the permission.
